src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+@import 'variables';
```

Comment:
no need to import it in every file
do it once in `main.scss` but above all other imports